ui/page/main_page.go

```diff
@@ -476,7 +478,17 @@ func (mp *MainPage) LayoutUSDBalance(gtx layout.Context) layout.Dimensions {
 	)
 }
 
+func (mp *MainPage) HideMyBalance(gtx layout.Context, hbal bool) layout.Dimensions {
+	if hbal {
+		hidenText := mp.Theme.Label(values.TextSize24, "********")
+		return hidenText.Layout(gtx)
+	}
+	return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
```

Comment:
this is only used on this page. no need to export it.
```suggestion
func (mp *MainPage) hideMyBalance(gtx layout.Context, hidden bool) layout.Dimensions {
	if hidden {
		return mp.Theme.Label(values.TextSize24, "********").Layout(gtx)
	}
	return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
```